Scripts/Swift/SetUpCICDService.swift

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/swift
+
+import Foundation
+
+let fileManager = FileManager.default
+
+enum CICDService {
+    case github, bitrise, codemagic, later, none
```

Comment:
I prefer to remove the `later` (and `none`), but this is still good!

Reply:
Updated https://github.com/nimblehq/ios-templates/pull/496/commits/d5bf8197b64aabd8fe16808ca480c48a6dd4ba01